networkit/cpp/clique/test/MaximalCliquesGTest.cpp

```diff
@@ -17,7 +17,10 @@ TEST_F(MaximalCliquesGTest, testMaximalCliques) {
 	Graph G = reader.read("input/hep-th.graph");
 
 	node seed = 2;
-	auto sn = G.neighbors(seed);
+    std::vector<node> sn;
+    sn.reserve(G.degree(seed));
+    for (node u : G.neighborRange(seed))
+        sn.push_back(u);
```

Comment:
`std::vector` has a constructor that takes iterators; this can be used to turn those 4 lines into a single line. The indentation seems to be off, too.

Reply:
This needed some fixes in the `NeighborIterator` and `NeighborWeightIterator` classes (9ce72ee); addressed in fbcc2b7.